DuplicateAssemblyScanner/DuplicateAssemblyScanner/Settings.cs

```diff
@@ -10,16 +10,32 @@ namespace DuplicateAssemblyScanner {
     /// Generates the settings screen based on cached results from the assembly scanner.
     /// </summary>
     public class Settings {
+        /// <summary>
+        /// Cache the number of duplicates per assembly name.
+        ///
+        /// Setting this to <c>null</c> causes a fresh scan next time
+        /// <c>Settings.CreateUI()</c> is called.
+        /// </summary>
+        internal static Dictionary<string, int> _duplicates;
```

Comment:
Non-readonly fields should not be `internal`, they should be `private`. If you want to expose this data, use a property.